src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER = 100;
+
+    public Ball getRandomBall() {
+        String color = new ColorSupplier().getRandomColor();
```

Comment:
ColorSupplier object should be class level variable

Reply:
But what if we use it only once? Is it still bad practice?